pkg/api/describe/describer.go

```diff
@@ -97,7 +98,8 @@ func describeBuilderImage(config *api.Config, image string, out io.Writer) {
 		Tag:                config.Tag,
 		IncrementalAuthentication: config.IncrementalAuthentication,
 	}
-	build.GenerateConfigFromLabels(c)
+	pr, _ := docker.GetBuilderImage(c)
```

Comment:
OK ... pushed update that does not ignore the err

On Fri, Aug 12, 2016 at 1:49 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/api/describe/describer.go
> <https://github.com/openshift/source-to-image/pull/570#discussion_r74631929>
> :
>
> > @@ -97,7 +98,8 @@ func describeBuilderImage(config *api.Config, image string, out io.Writer) {
> >  		Tag:                config.Tag,
> >  		IncrementalAuthentication: config.IncrementalAuthentication,
> >  	}
> > -	build.GenerateConfigFromLabels(c)
> > +	pr, _ := docker.GetBuilderImage(c)
>
> but now pr could be nil, in which case GenerateConfigFromLabels is going
> to error anyway, seems better to report the error as early as possible,
> otherwise it's going to get wrapped/lost (before the error from the pull
> got bubbled up as is)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/source-to-image/pull/570/files/adcdd6aa20b77e9c9ec5dce3b524cf20b22042ca#r74631929>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADbadDMZo_UdRKnFwa6WQRAsw1MDL31Rks5qfLIdgaJpZM4JjLrA>
> .
>